modules/chapter4/pages/section4.adoc

```diff
@@ -1,6 +1,191 @@
-= Install Sample Application on Tenant Cluster
+= Install and Configure Operators
 :experimental:
 
+In this section, you will be installing OpenShift Data Foundation operator on _Tenant_ cluster.
```

Comment:
It is one term. When we call just operator then it will be "an operator".

https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation/4.9/html/red_hat_openshift_data_foundation_architecture/openshift_data_foundation_operators#openshift-data-foundation-operator_rhodf